airbyte-integrations/connectors/source-coinmarketcap/source_coinmarketcap/spec.yaml

```diff
@@ -0,0 +1,36 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/coinmarketcap
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Coinmarketcap Spec
+  type: object
+  required:
+    - api_key
+    - data_type
+  additionalProperties: true
+  properties:
+    api_key:
+      title: API Key
+      type: string
+      description: >-
+        Your API Key. See <a
+        href="https://coinmarketcap.com/api/documentation/v1/#section/Authentication">here</a>. The token is
+        case sensitive.
+      airbyte_secret: true
+    data_type:
+      title: Data type
+      type: string
+      enum:
+        - latest
+        - historical
+      description: >-
+        /latest: Latest market ticker quotes and averages for cryptocurrencies and exchanges.
+        /historical: Intervals of historic market data like OHLCV data or data for use in charting libraries. See <a
+        href="https://coinmarketcap.com/api/documentation/v1/#section/Endpoint-Overview">here</a>.
+    symbol:
+      title: Symbol
+      type: array
```

Comment:
can you add a type for [the items in an array](https://json-schema.org/understanding-json-schema/reference/array.html#items)?

Reply:
Done!